website/docs/sql-function-glossary/sql-aggregate-functions/sql-avg.md

```diff
@@ -0,0 +1,51 @@
+---
+id: sql-avg
+title: SQL AVG
+description: The AVG function is used to straightforwardly calculate the average of a numeric column, but you may also see it used in a window function to calculate rolling averages.
+slug: /sql-function-glossary/sql-avg
```

Comment:
Curious what our rationale is for prepending each slug + title with `SQL`?  It reads as pretty verbose (especially in the slug) - see [BQ's docs](https://cloud.google.com/bigquery/docs/reference/standard-sql/json_functions) for a lower-SQL-density structure

<img width="280" alt="image" src="https://user-images.githubusercontent.com/77362861/204580064-4a476eaf-9a85-4ab7-9881-4dec5f96c3a3.png">

Reply:
The main purpose is to ensure that we are hitting the target keyword in the H1 and title tag. You're right though, we could remove it from the URL path for sure.